pkg/apis/directpv.min.io/v1beta1/volume.go

```diff
@@ -293,6 +293,16 @@ func (volume DirectPVVolume) IsSuspended() bool {
 	return string(volume.getLabel(types.SuspendLabelKey)) == strconv.FormatBool(true)
 }
 
+// SetUniqueAllocID sets the provided allocation id on the volume.
+func (volume *DirectPVVolume) SetUniqueAllocID(allocID string) {
+	volume.SetLabel(types.UniqueAllocIDLabelKey, types.LabelValue(allocID))
+}
+
+// GetUniqueAllocID gets the provided allocation id on the volume.
+func (volume *DirectPVVolume) GetUniqueAllocID() string {
+	return string(volume.getLabel(types.UniqueAllocIDLabelKey))
+}
```

Comment:
Not sure why we need to storage unique ID to volumes.

Reply:
because, upon releasing the corresponding unique volume, we have to remove the unique id from the drive.

Ref: https://github.com/minio/directpv/pull/823/files#diff-929b16adf1f9978f0d0981bc896b607a606e87a93bdb8b045be952c8b13916c1R190